client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -50,6 +51,7 @@ const DesignSelector: React.FunctionComponent = () => {
 
 	useTrackStep( 'DesignSelection', () => ( {
 		selected_design: getSelectedDesign()?.slug,
+		is_selected_design_premium: !! getSelectedDesign()?.premium,
```

Comment:
Is there a reason for not using the recently introduced [selector](https://github.com/Automattic/wp-calypso/pull/42377/files#diff-655ccf8d97a75836ef55d31cf1bbdf12R15)?

Reply:
Aye! I should totally use that. 👍